pkgs/top-level/python-packages.nix

```diff
@@ -7697,6 +7697,7 @@ in {
 
   rxv     = callPackage ../development/python-modules/rxv     { };
 
+  vulture = callPackage ../development/python-modules/vulture { };
```

Comment:
please try to add this in a semi-sorted fashion. Appending to the end will cause merge conflicts to others who do the same.

Reply:
Fixed in 0674589.